internal/connect.go

```diff
@@ -148,7 +147,7 @@ func ConnectToCluster(ctx context.Context, clientConfig *hazelcast.Config) (cli
 	defer cancel()
 	configCopy := clientConfig.Clone()
 	// prevent internal event loop to print error logs
-	configCopy.Logger.Level = logger.OffLevel
+	//configCopy.Logger.Level = logger.OffLevel
```

Comment:
I think we can move this to newly added "DefaultConfig". Wdyt?

Reply:
I had to comment out that line in order to be able to see the log. Probably `DefaultConfig` doesn't work as intended?

Reply:
You are correct! It should suffice if we can change the default log level here:
https://github.com/yuce/hazelcast-commandline-client/blob/ssl-config/internal/config.go#L67
`config.Logger.Level = logger.OffLevel`

This would initiate a default config on user system with client level logging disabled. Also we need to remove the line you correctly commented out above.

Sorry for this, I should have handled this better